app/javascript/Components/student_table.jsx

```diff
@@ -178,12 +196,19 @@ class RawStudentTable extends React.Component {
               Header: I18n.t("actions"),
               accessor: "_id",
               Cell: data => (
-                <span>
-                  <a href={Routes.edit_course_student_path(this.props.course_id, data.value)}>
-                    {I18n.t("edit")}
-                  </a>
-                  &nbsp;
-                </span>
+                <>
+                  <span>
+                    <a href={Routes.edit_course_student_path(this.props.course_id, data.value)}>
+                      {I18n.t("edit")}
+                    </a>
+                  </span>
+                  &nbsp;|&nbsp;
+                  <span>
+                    <a href="#" onClick={() => this.removeStudent(data.value)}>
+                      {I18n.t("delete")}
```

Comment:
Similar to my comment below: please add a new "Remove" string and use that instead (here and with TAs)